source/lexbor/core/lexbor.h

```diff
@@ -13,6 +13,10 @@ extern "C" {
 
 #include "lexbor/core/def.h"
 
+typedef void *(*lexbor_malloc_f)(size_t size);
```

Comment:
@zyc9012 

Naming.

typedef void *(*lexbor_memory_malloc_f)(size_t size);

and so on.